### PR TITLE
React2 week1

### PR DESCRIPTION
**Pull request by melo4yxa77796**
I have integrated the tasks from the React1 Week 3 module and the React2 Week 1 module to fetch and display meals in a grid format. I'm currently refining the styling and layout of the home page and other components.

### CODE REVIEW

**Comment by melo4yxa77796** [comment]
Screenshot 2024-11-01 at 18 22 42

File: app/frontend/components/MealsList.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+"use client";
```
Comment (markitosha) [inline app/frontend/components/MealsList.jsx]: You don't need to use `'use client'` here since you're using Vite instead of Next.js

**Review by markitosha — COMMENTED** [review]
Good job! No comments from me (almost) :)

File: app/frontend/components/MealsList.jsx (hunk 1 above)
Comment (melo4yxa77796) [reply app/frontend/components/MealsList.jsx]: 👍

**Pull request merged**